app/models/stream.rb

```diff
@@ -7,12 +7,13 @@ class Stream < ApplicationRecord
   friendly_id :name, use: %i[finders slugged scoped], scope: :organization
   belongs_to :organization
   has_many :uploads, dependent: :destroy
-  has_many :default_stream_histories, dependent: :destroy
+  has_one :default_stream_history, dependent: :destroy
   has_many :marc_records, through: :uploads, inverse_of: :stream
   has_many :files, source: :files_blobs, through: :uploads
   has_one :statistic, dependent: :delete, as: :resource
   has_many :normalized_dumps, dependent: :destroy
   has_many :job_trackers, dependent: :delete_all, as: :reports_on
+  has_many :interstream_deltas, dependent: :destroy
```

Comment:
I think this association is not needed (and may cause problems when an organization is deleted). Organizations already have a relationship to normalized_dumps and normalized_dumps have a relationship to interstream_deltas, so there is already a relationship between organizations & interstream_deltas through normalized_dumps.